### ISSUE TITLE
only output css on assets.outputCss

### ISSUE THREAD

**Issue by konsultaner** [phalcon/cphalcon#2408]
In my index.volt i used collections to output css an js like this

``` twig
{% if cssCollections is defined and cssCollections is iterable %}
    {% for cssCollection in cssCollections %}
         {{ assets.outputCss(cssCollection) }}
    {% endfor %}
{% endif %}
{% if jsCollections is defined and jsCollections is iterable %}
    {% for jsCollection in jsCollections %}
         {{ assets.outputJs(jsCollection) }}
    {% endfor %}
{% endif %}
```

My controller adds a javascript and a css file to the same collection

``` php
$this->assets->collection('index')->addCss('css/index.css');
$this->assets->collection('index')->addJs('js/ng/index/app.js');

$this->view->cssCollections = array("index");
$this->view->jsCollections = array("index");
```

I assumed that the asset manager is smart enough to output one link tag and one script tag but in stead I got this result:

``` html
<link rel="stylesheet" href="http://localhost/css/index.css" type="text/css">
<link rel="stylesheet" href="http://localhost/js/ng/index/app.js" type="text/css">
<script src="http://localhost/css/index.css" type="text/javascript"></script>
<script src="http://localhost/js/ng/index/app.js" type="text/javascript"></script>
```
##

**Comment by dancebear** [comment]
\Phalcon\Assets\Manager::output has parameter $type but not used

**Comment by sergeyklay** [comment]
Fixed in the `3.2.x branch`. Feel free to open new issue if the problem appears again. Thank you for contributing.
